lib/ui/src/components/sidebar/treeview/utils.ts

```diff
@@ -2,6 +2,8 @@ import memoize from 'memoizerific';
 import Fuse from 'fuse.js';
 import { DOCS_MODE } from 'global';
 import { SyntheticEvent } from 'react';
+import { State } from '@storybook/api';
+import { Root, Group, Story } from '@storybook/api/dist/lib/stories';
```

Comment:
`Root, Group, Story` should be exported at `@storybook/api` top level?

Reply:
yeah, agreed!